xml/tuning_sapconf.xml

```diff
@@ -609,16 +611,16 @@ Sapconf will not work properly!</screen>
     All parameters of &sapconf; can be found in the file
     <filename>/etc/sysconfig/sapconf</filename>.
     The file can be edited directly. All parameters in this file are
-    explained by means of comments and references to &sap; Notes which can
+    explained by means of comments and references to &sap; Notes, which can
     be viewed at <link xlink:href="https://launchpad.support.sap.com/"/>.
    </para>
    <para>
     When &sapconf; is updated, all customized parameters from this file will
     be preserved as much as possible. However, sometimes parameters cannot
     be transferred cleanly to the new configuration file. Therefore, after
     updating it is advisable to check the difference between the previous
```

Comment:
this sentence is a bit hard to read. Propose to remove therefore and just go "After updating, it is advisable..."